client/views/room/Header/QuickActions/QuickActions.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { memo, useContext, useCallback, useState, useEffect, useMemo } from 'react';
```

Comment:
I would move this files to some place like `client/views/room/Header/Omnichannel/QuickActions` as I already mentioned.  You can insert your header variation here if you want to